boefjes/tests/test_nmap.py

```diff
@@ -163,3 +169,18 @@ def test_nmap_arguments_udp_top250_ipv6(self):
             ],
             args,
         )
+
+    def test_normalizer(self):
+        input_ooi = parse_obj_as(
+            OOIType,
+            {
+                "object_type": "IPAddressV4",
+                "network": "Network|internet",
+                "address": "134.209.85.72",
+            },
+        )
+        boefje_meta = get_boefje_meta(input_ooi=input_ooi.reference)
+        boefje_meta.arguments["input"] = serialize_ooi(input_ooi)
+        output = [x for x in run(get_normalizer_meta(boefje_meta), get_dummy_data("raw/nmap_mispoes.xml"))]
+        self.assertEqual(17, len(output))
+        self.assertEqual("https", output[12].name)
```

Comment:
So rather than checking on an index, because it's harder to read/ maintain, you could just check for the existence of this object since we already know the list may contain only one instance of this time:
```suggestion
        self.assertEqual(1, len(list(filter(lambda x: x.name == "https", output)))
```

Reply:
Thanks, I changed the testing a bit to make it more generic.